v1.md

```diff
@@ -12,29 +12,31 @@ Here are examples of using the inline scanner in different pipelines.
 
 ## Minimum Requirements
 * Sysdig Secure > v2.5.0 access (with token)
+* Docker client with daemon running
 * Internet Access to post results to Sysdig Secure
 
 **Note**: For Airgapped environments, we suggest the following:
 
-* docker pull sysdiglabs/sysdig-inline-scan:latest (if using the inline scan container)
+* docker pull docker.io/anchore/anchore-engine:v0.8.1 (the version may differ based on the installation)
+* docker pull sysdiglabs/secure-inline-scan:latest (if using the inline scan container)
 * Open firewall settings to allow traffic to https://secure.sysdig.com/api/scanning
 
 **Note**: For onprem environments, use the -o flag in order to get the correct scan result URL.
 
 ## Usage
 
-Here below the different usages are described based on Sysdig installation type.
+Here below are described different usages based on Sysdig installation type.
 
 ### OnPrem
 
 #### Script
 ```
-curl -s https://download.sysdig.com/stable/inline_scan_docker.sh | bash -s -- -s <SYSDIG_REMOTE_URL> -o -k <TOKEN> <FULL_IMAGE_NAME>
+curl -s https://download.sysdig.com/stable/inline_scan.sh | bash -s -- analyze -s <SYSDIG_REMOTE_URL> -o -k <TOKEN> <FULL_IMAGE_NAME>
 ```
 
 #### Docker run
 ```
-docker run -v /var/run/docker.sock:/var/run/docker.sock sysdiglabs/sysdig-inline-scan:latest -s <SYSDIG_REMOTE_URL> -o -k <TOKEN> <FULL_IMAGE_NAME>
+docker run -v /var/run/docker.sock:/var/run/docker.sock sysdiglabs/secure-inline-scan:latest analyze -s <SYSDIG_REMOTE_URL> -o -k <TOKEN> <FULL_IMAGE_NAME>
```

Comment:
I know this is deprecated, but should we pin this to `secure-inline-scan:1`?

Reply:
V1 is the documentation for the v1, but secure-inline-scan:1 does not exist (it has been never used). So I would keep :latest pointing to v1 for know, and :2 pointing to the latest :2.x.x version.

The errors in this part of the V1.md (missing `analyze` and using sysdig-inline-scan instead of secure-inline-scan) are due to the fact that the V1.md was copied from the partially updated README.md in the V2 branch. I just restored the old V1 README.md from an older commit (from version 1.0.4).

Reply:
Understood. Deprecated docs explanations shouldn't work anyway.